packages/ml-natural-language/lib/validateTextMessage.js

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) 2016-present Invertase Limited & Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this library except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import { hasOwnProperty, isBoolean, isNumber, isObject, isString } from '@react-native-firebase/common';
+
+export default function validateTextMessage(textMessage) {
+  if (!isObject(textMessage)) {
+    throw new Error(
+      "'textMessage' expected an object value",
+    );
+  }
+
+  const out = {
+    timestamp: Date.now(),
+    isLocalUser: true,
+  };
+
+  if (!hasOwnProperty(textMessage, 'text')) {
```

Comment:
This isn't needed, the `isString` check below will return false if the property doesn't exist.